app/assets/javascripts/discourse/components/search-advanced-options.js.es6

```diff
@@ -228,7 +230,7 @@ export default Em.Component.extend({
 
     if (match.length !== 0) {
       const existingInput = _.isArray(tags) ? tags.join(',') : tags;
-      const userInput = match[0].replace(REGEXP_TAGS_PREFIX, '');
+      const userInput = match[0].replace(REGEXP_TAGS_PREFIX, '').replace(REGEXP_TAGS_SUFFIX, '');
```

Comment:
Not really... as that isn't how REGEXP_TAGS_PREFIX gets used and to set it up that way would mean special cases for it in all locations used. Plus, it is attempting to accomplish two separate goals targetting two separate parts of the string.

To make it work that way, I'd have to change REGEXP_TAGS_PREFIX and REGEXP_TAGS_SUFFIX to always be part of a concatenation so it would put the `/(` and `|` and `)/ig` in it when applicable. Versus calling replace chained.

The only alternative I could see is creating another const that contains both expressions for replace. I could look into that.